contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -1184,6 +1073,116 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         _removeUnusedSigningKeys(_nodeOperatorId, _fromIndex, _keysCount);
     }
 
+    function _isValidatorExitingKeyReported(bytes32 _keyHash) internal view returns (bool) {
+        return _validatorExitProcessedKeys[_keyHash];
```

Comment:
Maybe better this function receives `bytes pubKey` calculates hash and returns point in the map.
What happens if there is no such key in map?

Reply:
Let's rename to `isValidatoKeyReportedLate`

Reply:
make public